### DISCUSSION TITLE
Wyszukanie zamówienia po adresie email w domenie allegro

### DISCUSSION THREAD

**Discussion by bartekc7** [allegro/allegro-api#6668] [Suggestions - rejected]
Pomocne byłoby przeszukiwanie zasobu checkout-forms po parmetrze query z adresem email zamówienia w domenie allegro. 

https://developer.allegro.pl/documentation/#operation/getListOfOrdersUsingGET

**Comment by PrzemyslawLukanowski** [comment]
Dziękuję za sugestię. Obecnie jednak nie planujemy wprowadzać takiej zmiany, dlatego aktualizuję status sugestii na "odrzucona".
